src/components/DeliverableView/Description.tsx

```diff
@@ -1,10 +1,18 @@
 import { Box } from '@mui/material';
 import { ViewProps } from './types';
+import DeliverableStatusBadge from 'src/components/DeliverableView/DeliverableStatusBadge';
 
 const Description = (props: ViewProps): JSX.Element => {
+  const { deliverable, isAcceleratorConsole } = props;
+
   return (
     <Box display='flex' flexDirection='column'>
-      <div>Description coming soon {props.isAcceleratorConsole}!</div>
+      <div>
+        {deliverable.status !== 'Rejected' && !isAcceleratorConsole && (
```

Comment:
If you think it makes sense to combine the description and status bar components into one, we can do that. I created the placeholder components just as an initial draft. What you have here is fine too.

Reply:
Yeah I didn't love the logic being split across two components here either. I'm up for combining the two, any name ideas for the new combined component?

Reply:
`Metadata`?

Reply:
`DescriptionWithStatus`?

Reply:
Sure your suggestion is more descriptive and clear 👍

Reply:
I definitely prefer `Metadata`, thanks @karthikbtf 🙌